lib/suspenders/app_builder.rb

```diff
@@ -379,6 +379,22 @@ def init_git
       run 'git init'
     end
 
+    def git_init_commit
+      if @@user_choice.present? || @@user_choice.include?(:gitcommit)
+        run 'git add .'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.